speclang_modern.tex

```diff
@@ -2349,16 +2349,16 @@ \subsection{Memory blocks and pointer dereferencing}
 following built-in predicates deal with this notion:
 \begin{itemize}
 \item \valid\indexttbs{valid} applies to a set of terms 
-(see Section~\ref{sec:locations}) of some pointer type. 
+(see Section~\ref{sec:locations}) of some pointer type (different to \lstinline|void*|). 
```

Comment:
It is not necessary since the size of incomplete types can be specified into ACSL (i.e. by an axiom that has to be validated by the user), even if some Frama-C plug-ins do not support that.